src/transformers/models/bark/modeling_bark.py

```diff
@@ -1613,11 +1614,15 @@ def enable_cpu_offload(self, gpu_id: Optional[int] = 0):
         else:
             raise ImportError("`enable_model_cpu_offload` requires `accelerate`.")
 
-        device = torch.device(f"cuda:{gpu_id}")
+        device_type = "cuda"
+        if is_torch_accelerator_available():
+            device_type = torch.accelerator.current_accelerator().type
+        device = torch.device(f"{device_type}:{gpu_id}")
```

Comment:
I use `torch.accelerator` to detect device runtime when it's available, else fallback to old value which is "cuda".